CMakeLists.txt

```diff
@@ -465,3 +465,6 @@ set (CPACK_COMPONENT_DEVELOPER_DESCRIPTION "Include files")
 set (CPACK_COMPONENT_DOCUMENTATION_DESCRIPTION "OpenImageIO documentation")
 set (CPACK_COMPONENT_DEVELOPER_DEPENDS user)
 include (CPack)
+
+#REZ build
+include(RezBuild)
```

Comment:
As it is, this will break the build if it can't find this cmake file.

But I think what we can do is add

    option (BUILD_REZ "Build Rez support" OFF)

    if (BUILD_REZ)
        include(RezBuild)
    endif ()

and so on. Then we can iterate on the Rez support without interfering with anybody else.

Reply:
updated to use your suggestion.